src/platform/elements/form/Form.scss

```diff
@@ -259,7 +259,7 @@ novo-form {
                         >div.novo-control-outer-container {
                             display: flex;
                             align-items: center;
-                            max-width: 550px;
+                            max-width: 100%;
```

Comment:
UX wants this max-width in there. This would change all of Novo.

If you need it different in BBO, you will have to override it there.